src/main/java/htmlpublisher/HtmlPublisherTarget.java

```diff
@@ -98,13 +103,16 @@
      */
     @Deprecated
     public HtmlPublisherTarget(String reportName, String reportDir, String reportFiles, boolean keepAll, boolean allowMissing) {
-        this(reportName, reportDir, reportFiles, keepAll, false, allowMissing);
+        this(reportName, reportDir, reportFiles, keepAll, false, allowMissing, true);
     }
 
     public String getReportTitles() {
         return reportTitles;
     }
 
+    public HtmlPublisherTarget(String reportName, String reportDir, String reportFiles, boolean keepAll, boolean alwaysLinkToLastBuild, boolean allowMissing) {
```

Comment:
This constructor update shouldn't be needed (see comment on the `@DataBoundContructor` constructor below)

Reply:
corrected